Add missing Cryptography plugins to squeak builds

smalltalking opened the pull request:
DESPlugin, MD5Plugin and SHA2Plugin as external plugins

smalltalking committed:
Add missing Cryptography plugins to squeak builds

DESPlugin, MD5Plugin and SHA2Plugin as external plugins

fniephaus commented on build.linux32ARMv6/squeak.cog.spur/plugins.ext:
```diff
@@ -16,3 +16,6 @@ XDisplayControlPlugin \
 #GdbARMPlugin
+DESPlugin \
+MD5Plugin \
+SHA2Plugin \
```
Not sure these files are allowed to end with an ` \`. More importantly, these three new items must be listed *before* the first `#`. Currently, they are unfortunately all commented out. Yes, the syntax of this file is terrible!

smalltalking replied:
I wrote a script to rewrite this files en massse because there are so many of them. The script intentionally adds a backslash to the end of all non-comment lines because on linux it just works, and having the \ there has the benefit of easier addition of more lines in the future.
The interesting thing here is that only the 64-bit mac version had problem with the backslash. Neither linux, windows or the 32-bit mac had errors.

On linux, the commented out lines did not cause any problems either. I don't know which program processes these files (and raised the error), but the 64-bit mac version is clearly different that the ones on linux and windows which both seem to accept comments in between lines and the last line ending with \\.

smalltalking replied:
Moving the comments to the end turned out to be enough to fix the errors on 64-bit mac.

smalltalking committed:
Add missing Cryptography plugins to squeak builds part 2

Make sure that all commented out plugins in modified
plugins.ext files go to the end.
